resources/views/includes/header.blade.php

```diff
@@ -8,7 +8,7 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="/">{{ config('app.name', 'HRM') }}</a>
+      <a class="navbar-brand" href="/admin">{{ config('app.name', 'HRM') }}</a>
```

Comment:
Just so i don't leave a comment everywhere where you have made this note, all routes have been changes according to your request.